### ISSUE TITLE
Add tags to datasets

### ISSUE THREAD

**Issue by mfiume** [ga4gh/beacon-team#18]
Add arbitrary text tags that can be used to denote, for example, if the dataset is from case-level observations or curated annotation, whole exome or whole genome, population, geographic region, etc.

**Comment by mfiume** [comment]
Alternatively, we could add specific fields. I suggest at least the following:

type = { case-level, annotation, <any string> }
population = { <any string> }
scope = { genome , exome, <any string> }

**Comment by jrambla** [comment]
Are you suggesting a kind of "info" field for the dataset?
In positive +1

**Comment by mcupak** [comment]
Agreed to postpone to 1.0.

**Comment by mfiume** [comment]
This is becoming increasingly important as the number of beacons increases.

Also, beacons that serve in silico predictions respond "true" for most alleles. It should be made clear to beacon users that these are not really observed, but hypothetical. We get a lot of excited emails about hits on SolveBio's dbNSFP Beacon for example.

**Comment by david4096** [comment]
The G4 schemas have a string:List tag that is on the `Dataset` message for exchanging arbitrary data that is not in named fields.

https://github.com/ga4gh/schemas/blob/master/src/main/proto/ga4gh/metadata.proto#L45

**Comment by sdelatorrep** [comment]
@mfiume, can the "info" field already present in [BeaconDataset](https://github.com/ga4gh/beacon-team/blob/develop-proto/src/main/proto/ga4gh/beacon.proto) be used for this purpose (as @jrambla suggested)?

**Comment by mfiume** [comment]
Yes, just a matter of whether we'd like to standardize this so that networks can rely on consistent fields. WDYT @sdelatorrep?

**Comment by mbaudis** [comment]
While in personal use I'm all for "tags", a better way (when having the power to develop in a forward looking way) is to do this in a proper object way. This means:

* no simple "guess what they mean" tags
* no "named attribute for everything"

Similar to the (still not master but pretty much settled) "characteristics" attribute https://github.com/ga4gh/ga4gh-schemas/blob/metadata-integration/src/main/proto/ga4gh/bio_metadata.proto#L101

**Comment by sdelatorrep** [comment]
@mfiume we believe we should close this issue and open a new one when we have specific tags to include. Or you already have some?

**Comment by sdelatorrep** [comment]
Any new on this? Should we move the issue to another milestone? Or close it and reopen it when we eventually have some requirements?

**Comment by mbaudis** [comment]
@sdelatorrep Probably best to do this with concrete implementation. Also, evaluate then use of ontology terms (e.g. EFO) instead of arbitrary tags / typed values.

**Comment by susheel** [comment]
Suggest we move to next milestone - v1.0
Reason: Requirements for adding tags to datasets are not clear. No real-world use case to test.

**Comment by susheel** [comment]
Moving to next milestone v0.5
